src/java.base/share/classes/module-info.java

```diff
@@ -261,7 +261,8 @@
         jdk.internal.vm.ci,
         jdk.incubator.vector,
         jdk.jfr,
-        jdk.unsupported;
+        jdk.unsupported,
+        jdk.crypto.ec;
```

Comment:
jdk.crypto.ec has been hollowed out since JDK 22, the sun.security.ec are in java.base. So I don't think you need this qualified export.

Reply:
Thanks, fixed. (Started this when `jdk.crypto.ec` was still in use.. missed a few spots during rebase I guess)